disttask/loaddata/proto.go

```diff
@@ -29,29 +29,43 @@ const (
 
 // TaskMeta is the task of LoadData.
```

Comment:
What's the meaning of task/subtask/minimal task? For example, they correspond to data of some range?

Reply:
task: lightning task
subtask: chunks in splits by lightning engine
minitask: chunks in one engine